extensions/wikia/ARecoveryEngine/ARecoveryEngine.i18n.php

```diff
@@ -1,75 +1,93 @@
+	'arecovery-blocked-message-part-two' => 'Wikia is not accessible if you’ve made further modifications. Remove the custom ad blocker rule(s) and the page will load as expected.',
+);
+
+$messages['qqq'] = array(
+	'arecovery-blocked-message-headline' => 'Headline for message for ad blocked users',
+	'arecovery-blocked-message-part-one' => 'First part of message for ad blocked users',
+	'arecovery-blocked-message-part-two' => 'Second part of message for ad blocked users',
+);
+
+$messages['de'] = array(
+	'arecovery-blocked-message-headline' => 'Störung durch Adblocker erkannt!',
+	'arecovery-blocked-message-part-one' => 'Wikia ist eine gebührenfreie Seite, die sich durch Werbung finanziert. Benutzer, die Adblocker einsetzen, haben eine modifizierte Ansicht der Seite.',
+	'arecovery-blocked-message-part-two' => 'Wikia ist nicht verfügbar, wenn du weitere Modifikationen in dem Adblocker-Programm gemacht hast. Wenn du sie entfernst, dann wird die Seite ohne Probleme geladen.',
+);
+
+$messages['es'] = array(
+	'arecovery-blocked-message-headline' => '¡Interferencia de bloqueo de anuncios detectada!',
+	'arecovery-blocked-message-part-one' => 'Wikia es un sitio libre de uso que hace dinero de la publicidad. Contamos con una experiencia modificada para los visitantes que utilizan el bloqueo de anuncios',
+	'arecovery-blocked-message-part-two' => 'Wikia no es accesible si se han hecho aún más modificaciones. Si se quita el bloqueador de anuncios personalizado, la página cargará como se esperaba.',
+);
+
+$messages['fr'] = array(
+	'arecovery-blocked-message-headline' => "Interférence d'un bloqueur de publicité détectée !",
+	'arecovery-blocked-message-part-one' => "Wikia est un site gratuit qui compte sur les revenus de la publicité. L'expérience des lecteurs utilisant des bloqueurs de publicité est différente",
+	'arecovery-blocked-message-part-two' => "Wikia n'est pas accessible si vous avez fait d'autres modifications. Supprimez les règles personnalisées de votre bloqueur de publicité, et la page se chargera comme prévu.",
+);
+
+$messages['it'] = array(
+	'arecovery-blocked-message-headline' => 'Ad blocker interference detected!',
+	'arecovery-blocked-message-part-one' => 'Wikia is a free-to-use site that makes money from advertising. We have a modified experience for viewers using ad blockers',
+	'arecovery-blocked-message-part-two' => 'Wikia is not accessible if you’ve made further modifications. Remove the custom ad blocker rule(s) and the page will load as expected.',
+);
+
+$messages['ja'] = array(
+	'arecovery-blocked-message-headline' => '広告ブロッカーが検出されました。',
+	'arecovery-blocked-message-part-one' => '広告収入で運営されている無料サイトWikiaでは、このたび広告ブロッカーをご利用の方向けの変更が加わりました。',
+	'arecovery-blocked-message-part-two' => '広告ブロッカーが改変されている場合、Wikiaにアクセスしていただくことができなくなっています。カスタム広告ブロッカーを解除してご利用ください。',
+);
+
+$messages['pl'] = array(
+	'arecovery-blocked-message-headline' => 'Ad blocker interference detected!',
```

Comment:
Is this is what we've got from crowdin? Because I'm not that fluent in Polish but I'm pretty sure it's in English ;)